packages/ember/README.md

```diff
@@ -0,0 +1,73 @@
+<p align="center">
+  <a href="https://sentry.io" target="_blank" align="center">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
+  </a>
+  <br />
+</p>
+
+# Official Sentry SDK for Ember.js
+
+## Links
+
+- [Official SDK Docs](https://docs.sentry.io/quickstart/)
+- [TypeDoc](http://getsentry.github.io/sentry-javascript/)
+
+## General
+
+This package is an Ember addon that wraps `@sentry/browser`, with added functionality related to Ember. All methods available in
+`@sentry/browser` can be imported from `@sentry/ember`.
+
+### Installation
+
+As with other Ember addons, run:
+`ember install @sentry/ember`
+
+Then add the following config to `config/environment.js`
```

Comment:
What are the pros of allowing them to add to a config file if they are just going to end to calling something like `SentryForEmber()` anyway? Isn't it better to be explicit?

Reply:
Configuration is based on environment in Ember, conventionally. This way you don't have to do any environment checks when running app.js and can rely on the build tools to pass it in when building the bundle.

Reply:
Ok gotcha, that makes sense to me.